chromium_src/chrome/browser/ui/views/frame/browser_view.h

```diff
@@ -9,6 +9,7 @@
 #include "brave/browser/ui/brave_browser_window.h"
 #include "brave/browser/ui/views/bookmarks/brave_bookmark_bar_view.h"
 #include "brave/browser/ui/views/frame/brave_browser_view_layout.h"
+#include "brave/browser/ui/views/frame/brave_contents_layout_manager.h"
```

Comment:
Do we need to include in here? Seems like upstream just uses forward declarations in the header which `#define` below should cover. Can we move this include into the .cc to avoid header pollution?

Reply:
Good call. Done.